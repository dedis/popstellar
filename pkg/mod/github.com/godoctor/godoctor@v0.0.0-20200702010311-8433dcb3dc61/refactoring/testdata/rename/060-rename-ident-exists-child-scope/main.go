package main

import "fmt"

var hello = ":-(" // This is a different hello

// Test for renaming the local variable hello
func main() {
	hello = ":-)"  // Don't change this

	var hello string = "Hello"	// <<<<< rename,11,6,11,6,renamed,fail
          if true {
           renamed := "good"
           fmt.Println(renamed)
           fmt.Println(hello)         
           }  
	var world string = "world"	
	hello = hello + ", " + world
	hello += "!"
	fmt.Println(hello)
}
