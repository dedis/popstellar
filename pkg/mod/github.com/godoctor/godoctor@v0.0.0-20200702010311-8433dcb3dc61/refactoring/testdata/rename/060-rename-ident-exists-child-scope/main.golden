package main

import "fmt"

var hello = ":-(" // This is a different renamed

// Test for renaming the local variable renamed
func main() {
	hello = ":-)"  // Don't change this

	var renamed string = "Hello"	// <<<<< rename,11,6,11,6,renamed,fail
          if true {
           renamed := "good"
           fmt.Println(renamed)
           fmt.Println(renamed)         
           }  
	var world string = "world"	
	renamed = renamed + ", " + world
	renamed += "!"
	fmt.Println(renamed)
}
